code/modules/mob/living/living.dm

```diff
@@ -663,6 +662,7 @@ Sorry Giacom. Please don't be mad :(
 						G.assailant.Stun(2) //Temporarily stun the assailant to give the victim some fighting chance
 						qdel(G)
 			if(resisting)
+				last_special = world.time + CLICK_CD_BREAKOUT //Additional cooldown
```

Comment:
This means that it will only add to the resist cooldown if you did actually resist or attempt to resist out of grabs.
Fixes resisting buckles.